src/scripts/main.js

```diff
@@ -1,3 +1,15 @@
 'use strict';
 
-// write your code here
+const populations = document.querySelectorAll('span.population');
+const totalPopulation = document.querySelector('span.total-population');
+const averagePopulation = document.querySelector('span.average-population');
+
+const sum = [...populations].reduce(
+  (prev, amount) => prev + (+amount.textContent.replaceAll(',', '') || 0),
```

Comment:
The use of `replaceAll` might not be supported in all JavaScript environments, especially older ones. Consider using `replace` with a regular expression `/,/g` to ensure broader compatibility.